File: тз/ColaFactory.java
package ru.agentd;

public class ColaFactory implements SodaFactory {
    @Override
    public Bottle createBottle() {
        return new ColaBottle();
    }

    @Override
    public Soda createSoda() {
        return new ColaSoda();
    }

    @Override
    public Label createLabel() {
        return new ColaLabel();
    }

    @Override
    public Closer createCloser() {
        return new ColaCloser();
    }
}

File: ColaFactory.ts
class ColaFactory implements SodaFactory {
    //Override
    public createBottle(): Bottle {
        return new ColaBottle();
    }

    //Override
    public createSoda(): Soda {
        return new ColaSoda();
    }

    //Override
    public createLabel(): Label {
        return new ColaLabel();
    }

    //Override
    public createCloser(): Closer {
        return new ColaCloser();
    }
}